src/web/demo.rkt

```diff
@@ -52,6 +52,8 @@
    [("api" "calculate") #:method "post" calculate-endpoint]
    [("api" "cost") #:method "post" cost-endpoint]
    [("api" "mathjs") #:method "post" ->mathjs-endpoint]
+  ;  added endpoint API rule -Ben
```

Comment:
We don't need this comment, it's clear from the commit!

Reply:
Resolved!